src/Components/MainPage/MainPage/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I believe this page should not exist. The Home page should be the list of open positions.

Reply:
Well it depends on the group decision. It's not so bad to have a welcome page